console/billing/how-to/use-the-cost-manager.mdx

```diff
@@ -0,0 +1,58 @@
+---
+meta:
+  title: How to use the billing cost manager
+content:
+  h1: How to use the billing cost manager
+  paragraph: This page explains how to use the billing cost manager
+tags: billing budget cost-manager
+dates:
+  validation: 2024-03-26
+  posted: 2024-03-26
+categories:
+  - billing
+---
+
+The Scaleway cost manager is a comprehensive tool to monitor and analyze cloud resource consumption and costs. It offers a clear and detailed view of consumption data, enabling you to accurately reconcile costs for re-invoicing purposes.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
+    </Message>
+2. Click the **Consumption** tab.
+3. Scroll to **Cost Manager**.
+
+    An overview of your Organization's consumption is displayed.
+
+4. Filter the data by selecting the different categories in the drop-down, then clicking **Filter**.
+
+    You can filter by:
+      - **Period** - Currently you can filter by year.
+      - **Category** - The product category (e.g. Compute, Storage, Containers).
+          <Message type="note">
+            If you are filtering by Product, you must first select the Product Category.
+          </Message>
+      - **Product** - The product itself. The consumption shown is the sum of all your products of the same type. For example, if you select **Instances**, the consumption chart will display the value referring to all Instances in your selected project. Each product corresponds to a different color in the consumption chart and in the detailed list.
```

Comment:
Each **category** corresponds to a different color in the consumption chart and in the detailed list.